umapi_client/functional.py

```diff
@@ -162,10 +115,7 @@ def create(self, first_name=None, last_name=None, country=None, email=None,
             if not self.email:
                 raise ArgumentError("You must specify email when creating a user")
         elif self.email is None:
-            self._validate(email=email)
             self.email = email
-        elif self.email.lower() != email.lower():
```

Comment:
I cannot figure out for the life of me what this test was supposed to be doing, and how these two could ever differ.  So I'm glad you're taking it out, but the fact that I don't know why it's here really raises a red flag for me: removing lines that don't seem to do anything usually means we just don't understand why there were put there.  I would go back in history to find out when this line was added to see if it was to fix some very obscure bug (e.g., when this call is invoked from somewhere else, or even recursively in this function).

Reply:
The only time `email` is used in this function is to initialize `self.email` if it is None.  That check just ensures that the `email` param is set to the same value as `self.email` (assuming `self.email` is not None).  Setting `email` to a different value won't actually do anything in this case, since `self.email` is used in the create command construction.

Now that I've thought more about it, it doesn't hurt to keep the check.  It could at least reduce confusion and ensure consistency.  I'm inclined to add it back.